test/seed.js

```diff
@@ -480,7 +485,7 @@ describe("Contract: Seed", async () => {
 
             await alternativeSetup.seed
                 .connect(admin)
-                .addClass(1e14, 1e12, 1e12, 10000000);
+                .addClass(1e14, 1e12, 1e12, CLASS_VESTING_DURATION, CLASS_VESTING_START_TIME);
```

Comment:
I think you forgot to change 1e14, 1e12 to constants, or just didn't push the commit...

Reply:
they were pushed into 19 branch, as for those time this and 19 branches weren't connected... 
added only them here also